backend/dataall/core/environment/services/environment_service.py

```diff
@@ -409,6 +417,22 @@ def invite_group(uri, data=None) -> (Environment, EnvironmentGroup):
                 resource_type=Environment.__name__,
             )
 
+            # Attach the organization policy to the group, if group hasn't it yet
+            if not ResourcePolicyService.check_group_resource_permission(
+                session=session,
+                group=group,
+                resource_uri=environment.organizationUri,
+                resource_type=Organization.__name__,
+                permission_name=GET_ORGANIZATION,
+            ):
+                ResourcePolicyService.attach_resource_policy(
+                    session=session,
+                    group=group,
+                    resource_uri=environment.organizationUri,
+                    permissions=[GET_ORGANIZATION],
+                    resource_type=Organization.__name__,
+                )
```

Comment:
The way `def attach_resource_policy()` is written is that the policy is only created and permissions associated if and only if they do not already exist (like a "create if not exists" expressions)

Open ended Q - is there added benefit to having the check `if check_group_resource_permission()`?

Reply:
True, attach_resource_policy does not create duplicates if the policy already exists. I don't think we need the check at creation, only at deletion

Reply:
done